org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/services/GenericEndpoint.java

```diff
@@ -28,6 +29,7 @@
 public class GenericEndpoint implements Endpoint {
 	
 	private static final Logger LOG = Logger.getLogger(GenericEndpoint.class.getName());
+	private static Object[] NO_ARGUMENTS = {};
```

Comment:
It is a minor thing, and `NO_ARGUMENTS` an array anyway, but still, could we make this final?